airgun/views/host.py

```diff
@@ -127,7 +127,7 @@ def after_fill(self, was_change):
 
     @property
     def is_displayed(self):
-        return self.browser.wait_for_element(
+        return self.root_browser.wait_for_element(
```

Comment:
This line is breaking my understanding of this browser object, I was thinking that the same instance of browser is passed around when initially created, so when View is created should have an instance of browser of parent that created it (the view creator will pass that instance), and now I am thinking of two different kinds of browsers? please explain?
even that we use root_parent , was thinking that it's the same as browser eg: self.browser ==  self.root_browser
or we have two different browser objects.
I am always thinking if they are different => we have a bug.

Reply:
I saw the Tab widget, and root_browser  is used, as the browser return a Wrapper object, this issue is not part of airgun, so ACK